src/paths.jl

```diff
@@ -23,14 +15,32 @@ function path!{T<:Colorant}(img::AbstractArray{T, 2}, vertices::Array{CartesianI
 
     for i in 1:length(vertices)-1
         if min(f,vertices[i+1])==f && max(l,vertices[i+1])==l
-            line!(img, vertices[i], vertices[i+1], color)
+            draw!(img, LineSegment(vertices[i], vertices[i+1]), color)
         else
             println(vertices[i+1])
             error("Point coordinates out of range.")
         end
     end
+end
 
-    if closed==true
-        line!(img, vertices[1], vertices[end], color)
-    end
-end
\ No newline at end of file
+#Polygon methods
+
+Polygon(v::Vector{Tuple{Int, Int}}) = Polygon([Point(p...) for p in v])
+Polygon(v::Vector{CartesianIndex{2}}) = Polygon([Point(p) for p in v])
```

Comment:
Two orthogonal comments:

- Do these result on the same as `Polygon.(v)` ? Should we provide explicit vectorized functions if broadcast would do the trick?

- If we do provide these, dispatching on `AbstractVector` would be a little more flexible